### PR TITLE
Updating CallRecords API v1.0 documentation - New fields added

### PR DESCRIPTION
**Pull request by Datar89**
We're including 8 new properties to the current response. We're not changing any behavior as part of these changes. The new properties that you will find are:

- New property for microsoft.graph.callRecords.session
    - isTest: Flag to indicate if the session corresponded to a session within test call.
  
- New properties for microsoft.graph.callRecords.participantEndpoint
    - name: Name of the device used by the media endpoint.
    - cpuName: CPU name used by the media endpoint.
    - cpuCoresCount: CPU cores count for the device used by the media endpoint.
    - cpuProcessorSpeedInMhz: CPU processor speed for the device used by the media endpoint.

- New properties for microsoft.graph.callRecords.mediaStream
    - rmsFreezeDuration: Root mean square for the received freeze duration related with the video stream.
    - averageFreezeDuration: Average for the received freeze duration related with video stream.
    - isAudioForwardErrorCorrectionUsed: Flag to indicate if the forward error correction (FEC) was used at some point in the call session.

**As part of this PR, we're updating** the examples for our API and definitions of the entities where the new fields were defined.

----------------------------------------------------------------

Roadmap item: Feature 16881 https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/16881

Beta Documentation PR: https://github.com/microsoftgraph/microsoft-graph-docs/pull/20859

### CODE REVIEW

**Review by mcm223 — APPROVED** [review]

**Review by GageAmes — APPROVED** [review]

File: api-reference/v1.0/resources/callrecords-mediastream.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Represents information about a media stream between two endpoints in a call.
 |averageAudioDegradation|Double|Average Network Mean Opinion Score degradation for stream. Represents how much the network loss and jitter has impacted the quality of received audio.|
 |averageAudioNetworkJitter|Duration|Average jitter for the stream computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
 |averageBandwidthEstimate|Int64|Average estimated bandwidth available between two endpoints in bits per second.|
+|averageFreezeDuration|Duration|Average of the received freeze duration related to the video stream.|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/callrecords-mediastream.md:24]:
```suggestion
|averageFreezeDuration|Duration|Average duration of freezing in the video stream.|
```

File: api-reference/v1.0/resources/callrecords-mediastream.md [hunk 2]
```diff
@@ -39,6 +41,7 @@ Represents information about a media stream between two endpoints in a call.
 |maxRoundTripTime|Duration|Maximum network propagation round-trip time computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
 |packetUtilization|Int64|Packet count for the stream.|
 |postForwardErrorCorrectionPacketLossRate|Double|Packet loss rate after FEC has been applied aggregated across all video streams and codecs.|
+|rmsFreezeDuration|Duration|Root mean square of the received freeze duration related to the video stream.|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/callrecords-mediastream.md:44]:
```suggestion
|rmsFreezeDuration|Duration|Root mean square of the average video stream freezing duration.|
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 3]
```diff
@@ -537,8 +537,92 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the `averageFreezeDuration` property to the [mediaStream](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-mediaStream?view=graph-rest-1.0) resource.",
+                    "Target": "mediaStream"
+                },
+                {
+                    "Id": "234eb8d2-60a4-4131-b848-88de56251792",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "isAudioForwardErrorCorrectionUsed",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `isAudioForwardErrorCorrectionUsed` property to the [mediaStream](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-mediaStream?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:558]:
```suggestion
                    "Description": "Added the **isAudioForwardErrorCorrectionUsed** property to the [mediaStream](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-mediaStream?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 4]
```diff
@@ -537,8 +537,92 @@
...
+                    "Target": "mediaStream"
+                },
+                {
+                    "Id": "234eb8d2-60a4-4131-b848-88de56251792",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "rmsFreezeDuration",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `rmsFreezeDuration` property to the [mediaStream](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-mediaStream?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:566]:
```suggestion
                    "Description": "Added the **rmsFreezeDuration** property to the [mediaStream](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-mediaStream?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 5]
```diff
@@ -537,8 +537,92 @@
...
+            "SubArea": "Calls and online meetings"
+          },
+          {
+            "ChangeList": [
+                {
+                    "Id": "d8a9644c-28b9-4125-83bc-c1c0138c1744",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "cpuCoresCount",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `cpuCoresCount` property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:584]:
```suggestion
                    "ChangeType": "Addition",
                    "Description": "Added the **cpuCoresCount** property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 6]
```diff
@@ -537,8 +537,92 @@
...
+                    "Target": "participantEndpoint"
+                },
+                {
+                    "Id": "d8a9644c-28b9-4125-83bc-c1c0138c1744",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "cpuName",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `cpuName` property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:592]:
```suggestion
                    "Description": "Added the **cpuName** property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 7]
```diff
@@ -537,8 +537,92 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the `cpuProcessorSpeedInMhz` property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
+                    "Target": "participantEndpoint"
+                },
+                {
+                    "Id": "d8a9644c-28b9-4125-83bc-c1c0138c1744",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "name",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `name` property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:608]:
```suggestion
                    "Description": "Added the **name** property to the [participantEndpoint](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-participantEndpoint?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IC3.DataPlatform.json [hunk 8]
```diff
@@ -537,8 +537,92 @@
...
+                    "Target": "participantEndpoint"
+                },
+                {
+                    "Id": "d8a9644c-28b9-4125-83bc-c1c0138c1744",
+                    "ApiChange": "Property",
+                    "ChangedApiName": "isTest",
+                    "ChangeType": "Addition",
+                    "Description": "Added the `isTest` property to the [session](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-session?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.IC3.DataPlatform.json:616]:
```suggestion
                    "Description": "Added the `isTest` property to the [session](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-session?view=graph-rest-1.0) resource.",
```
```suggestion
                    "Description": "Added the **isTest** property to the [session](https://learn.microsoft.com/en-us/graph/api/resources/callRecords-session?view=graph-rest-1.0) resource.",
```

**Review by Danielabom — APPROVED** [review]
@Datar89, thank you for working on this PR. PR approved.

File: api-reference/v1.0/resources/callrecords-mediastream.md [hunk 9]
```diff
@@ -39,6 +41,7 @@ Represents information about a media stream between two endpoints in a call.
 |maxRoundTripTime|Duration|Maximum network propagation round-trip time computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
 |packetUtilization|Int64|Packet count for the stream.|
 |postForwardErrorCorrectionPacketLossRate|Double|Packet loss rate after FEC has been applied aggregated across all video streams and codecs.|
+|rmsFreezeDuration|Duration|Root mean square of the average video stream freezing duration.|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/callrecords-mediastream.md:44]:
```suggestion
|rmsFreezeDuration|Duration|Average duration of the received freezing time in the video stream represented in root mean square.|
```

**Pull request merged**